mail/templates/courseEnrollmentEmail.js

```diff
@@ -0,0 +1,87 @@
+exports.courseEnrollmentEmail = (courseName, name) => {
+    return `<!DOCTYPE html>
```

Comment:
oh wait! what? are we not using html files to do this? wrong bro

Reply:
Haven't been through that part yet!. I mean haven;t learnt that part ever

Reply:
I guess using HTML file and using html file to string parser will get the best practice

Reply:
ok